Sources/Analytics/Configuration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Configuration.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 26/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+extension Analytics {
+    
+    /// Anaylitics configuration.
+    /// Used to configure values in the `Analytics`
+    /// 💣 QueueQoS 👉 The qos to be used by the analytics queue
```

Comment:
Not sure about these emoji in the docs 😇 

Do they appear correctly when Alt+Clicking?